lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -46,7 +47,7 @@ <h3 class="accomplishment-signatories-title sr-only">${_("Noted by")}</h3>
                             % if certificate_data:
                                 % for signatory in certificate_data.get('signatories', []):
                                 <div class="signatory">
-                                    <img class="signatory-signature" src="${static.url(signatory['signature_image_path'])}" alt="${signatory['name']}">
+                                    <img class="signatory-signature" src="${static.url(signatory['signature_image_path']) | n, decode.utf8}" alt="${signatory['name']} | n, decode.utf8">
```

Comment:
I think the last call to `| n, decode.utf8` may have been placed there in error? It looks like it's part of the HTML and not the mako code.

Reply:
Yep its copy/paste error. I'll fix that.

Reply:
We still need to apply HTML escaping to the img source URL. Please remove the n filter. See http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/preventing_xss.html#url-context-and-escaping for more details.